x-pack/solutions/security/plugins/security_solution/common/endpoint/types/custom_scripts.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Represents a custom script unified fields used in Scripts picker
+ */
+export interface CustomScript {
```

Comment:
Is this API response structure?

if so, please change it to use our "standard" format an API that returns a list of items:

```
{
    data: CustomScript[]
}
``